### PR TITLE
Switch to Xen v4.14 release

### PR DESCRIPTION
**Pull request by otyshchenko1**
The one important thing here is that now EXPERT mode can be selected
from the menuconfig directly.
There is no need to use XEN_CONFIG_EXPERT=y on the make command line
anymore.

Also install xenhypfs tool.

Signed-off-by: Oleksandr Tyshchenko <oleksandr_tyshchenko@epam.com>

### CODE REVIEW

**Comment by rshym** [comment]
I refrain from review due to lack of knowledge.

**Comment by otyshchenko1** [comment]
Preliminary tested on Prod-devel with DomU and DomA. No issues were observed.

File: inc/xen-version.inc [hunk 1]
```diff
@@ -3,6 +3,6 @@
 # Following inc file defines XEN version and needed glue info to get it
 # built with the current yocto version
 ################################################################################
-require ../meta-xt-images-domx/recipes-extended/xen/xen-4.13-thud.inc
+require ../meta-xt-images-domx/recipes-extended/xen/xen-4.14-thud.inc
```
Comment (andr2000) [inline inc/xen-version.inc:6]: @iusyk this is the dependency on the other external layer.
Just imagine that we have meta-xt-images-domx not at "../" location, but somewhere else and that will break
Beside that using an inc file from the other meta-layer looks odd.

Comment (iusyk) [reply inc/xen-version.inc:6]: I am working with this

Comment (iusyk) [reply inc/xen-version.inc:6]: > @iusyk this is the dependency on the other external layer.
> Just imagine that we have meta-xt-images-domx not at "../" location, but somewhere else and that will break
> Beside that using an inc file from the other meta-layer looks odd.

Frankly, I can not understand why xen-4.14-thud.inc belongs to meta-xt-images - this file is not used there. I think it is reasonable move this file in product. Any objections ?

Comment (otyshchenko1) [reply inc/xen-version.inc:6]: many things in meta-xt-images are not used by it directly, it is intended to contain common for all products stuff, I think.

**Comment by otyshchenko1** [comment]
We decided to merged it as is for now.

**Pull request merged**